libdokan/folderlist.go

```diff
@@ -86,7 +93,15 @@ func (fl *FolderList) open(ctx context.Context, oc *openContext, path []string)
 		}
 		fl.mu.Unlock()
 
+		h, er := libkbfs.ParseTlfHandle(
```

Comment:
I see. I'm thinking maybe I can move this call back to where it was (right before the switch-case), and use `name` (or `req.name` in `libfuse`) here instead. But this assumes that `fl.folders` never contains non-Canonical names. Do you think that's a safe assumption?